cron.sh

```diff
@@ -1,11 +1,26 @@
 #!/bin/bash
-set -euo pipefail # See http://redsymbol.net/articles/unofficial-bash-strict-mode/
-IFS=$'\n\t'
-
+# See http://redsymbol.net/articles/unofficial-bash-strict-mode/
+# explicitly without "-e" for it should not exit immediately when failed but write a mail
```

Comment:
We don't have to disable the `-e` for the entire script, see http://redsymbol.net/articles/unofficial-bash-strict-mode/#expect-nonzero-exit-status

Reply:
> This short-circuiting with the boolean operator makes the expression inside $( ... ) always evaluate successfully.

This would not be good - note the test of the exit status  in row 18 https://github.com/hbz/lobid-gnd/pull/180/files#diff-f754b24d9747c7dc4c61cd7b6b2636ccR18.

Reply:
Yes, I think the second suggestion from http://redsymbol.net/articles/unofficial-bash-strict-mode/#expect-nonzero-exit-status applies here.

Reply:
So : we initially set `set -e` just to add `set +e; $code ; set -e` ? Don't think this makes much sense but bloat and deceives. Also, other than stated at the beginning of redsymbol, `set -e` makes exception-handling impossible.
From the accepted answer of https://stackoverflow.com/questions/19622198/what-does-set-e-mean-in-a-bash-script:
>   -e  Exit immediately if a command exits with a non-zero status.
But it's considered bad practice by some people (bash FAQ and people writing this FAQ on irc freenode #bash) say it's better use:
trap 'do_something' ERR

Of course my main argument is given in the headline of the redsymbol-page:
> Use the Unofficial Bash Strict Mode (Unless You Looove Debugging)

you know I looove debugging ;)

Reply:
> we initially set set -e just to add set +e; $code ; set -e ?

But we would set that only for the command where we expect non-0, the rest would still be safer.

> you know I looove debugging ;)

Well, +1 then.